Feature: Calendar for multishipping and removed Show More in single shipping

azeeshan1039 opened the pull request:
## What?
It contains modifications for the multi shipping checkout shipping options with a Calendar and the removal of "Show More" for single shipping scenarios.

azeeshan1039 committed:
Feature: Calendar for multishipping and removed Show More in single shipping

wojciech29 commented on packages/core/src/app/shipping/shippingOption/MultiShippingOptions.tsx:
```diff
@@ -5,52 +5,59 @@ import { TranslatedString } from '@bigcommerce/checkout/locale';
-    isLoading: boolean;
-    shippingQuoteFailedMessage: string;
-    resetErrorConsignmentNumber(): void;
+  consignment: Consignment;
```
@azeeshan1039 

- there are changes like this which are unnecessary (it's only a change of indent)
- please avoid these changes, we would like to keep our changes to BC upstream version as minimal as possible

azeeshan1039 replied:
I have fixed the formatting issues in my latest commit.

wojciech29 commented on packages/core/src/app/shipping/shippingOption/MultiShippingOptions.tsx:
```diff
@@ -5,52 +5,59 @@ import { TranslatedString } from '@bigcommerce/checkout/locale';
 import { useCheckout } from '@bigcommerce/checkout/payment-integration-api';
 import { Alert, AlertType } from '@bigcommerce/checkout/ui';
 
+// import CalendarShippingOptions from './CalendarShippingOptions';
```
@azeeshan1039 

- please clean up the code from commented out code

azeeshan1039 replied:
Comments removed

wojciech29 reviewed (changes requested):
@azeeshan1039 

- please do not include any generated file in this PR
- generating those files will be part of our post merge pipeline

azeeshan1039 commented:
> @azeeshan1039
> 
> * please do not include any generated file in this PR
> * generating those files will be part of our post merge pipeline

- Master contains the dist directory. Do I need to ignore both dist and build during push? Or only the build files.

wojciech29 commented:
@azeeshan1039 

> Do I need to ignore both dist and build during push? Or only the build files.

- yes please, such changes will break our auto-sync with BC upstream

azeeshan1039 committed:
Feature: Calendar Shipping Option - Code Fixes

azeeshan1039 commented:
> @azeeshan1039
> 
> > Do I need to ignore both dist and build during push? Or only the build files.
> 
> * yes please, such changes will break our auto-sync with BC upstream

I have removed both of those folders and fixed the issues you highlighted in your comments. Please take a look and let me know.
